lib/api/apiUtils/object/coldStorage.js

```diff
@@ -28,6 +33,99 @@ function getAmzRestoreResHeader(objMD) {
     return undefined;
 }
 
+
+/**
+ * Check if restore can be done.
+ *
+ * @param {BucketInfo} bucketMD - bucket instance
+ * @param {ObjectMD} objectMD - object metadata
+ * @return {ArsenalError|boolean} - undefined if the conditions for RestoreObject are fulfilled
+ */
+function _validateStartRestore(bucketMD, objectMD) {
```

Comment:
Adding some error logs might help to debug.

Reply:
done https://github.com/scality/cloudserver/pull/4507/commits/4bf88225092246150b1f22c1fba17a36d6b0a5f1